src/content/ja/2019/accessibility.md

```diff
@@ -0,0 +1,418 @@
+---
+part_number: II
+chapter_number: 9
+title: アクセシビリティ
+description: 読みやすさ、メディア、操作性の容易さ、および支援技術とその互換性をカバーする2019 Web Almanacアクセシビリティの章。
+authors: [nektarios-paisios, obto, kleinab]
+reviewers: [ljme]
+translators: [MSakamaki]
+discuss: 1764
+published: 2019-11-11T00:00:00.000Z
+last_updated: 2019-12-30T00:00:00.000Z
+---
+
+## 導入
+
+Webのアクセシビリティは、包摂的で公平な社会の上では無くてはならない存在です。
+私たちの社会性と仕事や生活の多くがオンラインの世界に推移するにつれて、障害のある人々も分け隔てなく、すべてのオンラインの対話に参加できることがさらに重要になってきます。
+建築家が車椅子用の傾斜路のようなアクセシビリティ機能を作成や省略できるように、Web開発者はユーザーが宛にしている支援技術で助けたり邪魔をしたりできます。
+
+障害を持つユーザーの事を考えた時ユーザージャーニーはほぼ同じとなることを忘れないでください、彼らは異なるツールを使っているだけでしかありません。
+よく知られてるツールとして、スクリーンリーダー、画面拡大鏡、ブラウザまたは文字の拡大、音声コントロールなどがありますが、これ以外にも色々とあります。
+
+ほとんどの場合、アクセシビリティを改善することでサイトを訪れるすべての人に対してメリットを与える事ができます。
+私達は普通、障害者は生涯その障害を抱えていると思っていますが、一時的だったり状況的に障害を持つような人も居ます。
+たとえばその誰かが全盲なのか、一時的な目の感染症なのか、はたまた野外で眩しい太陽の下という状況なのか。
+これらすべて、その誰かが画面を見ることができない理由の説明になります。
+誰もが状況により障害を持ちうるため、Webページのアクセシビリティを改善することは、あらゆる状況ですべてのユーザーの体験を向上させることに繋がります。
+
+[Web Content Accessibility Guidelines](https://www.w3.org/WAI/WCAG21/quickref/) (WCAG)はWebサイトの利便性を向上する方法についてのアドバイスが纏められています。
```

Comment:
Should the title of this text not be translated? And maybe a note added that the links is in English?

Reply:
As you say, a lot of Japanese can be found by searching, so I will translate.